ui/app/mainui/AppMain.qml

```diff
@@ -64,6 +65,8 @@ Item {
     property NetworkConnectionStore networkConnectionStore: NetworkConnectionStore {}
     property CommunityTokensStore communityTokensStore: CommunityTokensStore {}
     property CommunitiesStore communitiesStore: CommunitiesStore {}
+    property WalletStore.TokensStore tokensStore: WalletStore.RootStore.tokensStore
```

Comment:
could be `readonly`, similarly stores above

Reply:
Yes!! I'll update it. I'll not touch the rest, only focusing on the `TokensStore` right now.